pkg/api/metrics_response.go

```diff
@@ -0,0 +1,100 @@
+package api
+
+import (
+	"sort"
+
+	jsoniter "github.com/json-iterator/go"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+type metricsResponse struct {
+	Results results
+}
+
+type results map[string]queryResponse
```

Comment:
why not just use [QueryDataResponse](https://github.com/grafana/grafana-plugin-sdk-go/blob/main/backend/data.go#L75)

or is this just part of the draft?

Reply:
interesting -- seems odd to have a few parallel representations of the same thing.  In particular when the real concern from @kylebrandt etc is that the errors are not accessible to consumers anyway.  Injecting the errors directly in the json response means we both do not have a standard way to read or write it from any consumer